install-template.sh

```diff
@@ -4,19 +4,25 @@
 
 set -eu
 
+@@additionalPrefix@@
+
 test_nix_installation () {
   nix --version 1> /dev/null 2> /dev/null
 }
 
 install_nix () {
   TMP=$(mktemp -d)
-  echo "extra-substituters = https://cache.garnix.io" >> "$TMP/nix-extra-config"
-  echo "extra-trusted-public-keys = cache.garnix.io:CTFPyKSLcx5RMJKfLo5EEPUObbA78b0YQ2DTCJXqr9g=" >> "$TMP/nix-extra-config"
 
-  curl --proto '=https' --tlsv1.2 -sSfL https://releases.nixos.org/nix/nix-2.17.1/install -o "$TMP/install.sh"
-  chmod u+x "$TMP/install.sh"
+  curl @@forceHttpsOption@@ --tlsv1.2 -sSfL @@baseUrl@@/nix-installer.sh -o "$TMP/nix-installer.sh"
+  chmod u+x "$TMP/nix-installer.sh"
 
-  "$TMP/install.sh" --nix-extra-conf-file "$TMP/nix-extra-config" --daemon --yes
+  export NIX_INSTALLER_BINARY_ROOT=@@baseUrl@@
+  "$TMP/nix-installer.sh" \
+    install \
+    --no-confirm \
```

Comment:
Do we want `--no-confirm` here? The zero-to-nix installer has much better output and asks much better questions. I'd say we don't.